G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/GenericMappingTools/gmt/releases/download/$(version)/gmt-$(version)-src.tar.gz",
+                  "ab7062912aeead1021770fad4756e0a99860fde8ea9b428fb00c22fa15a3bbfc"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+if [[ "${target}" == *-mingw* ]]; then
+    for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+        atomic_patch -p1 ${f}
+    done
+fi
+cd gmt-6.2.0
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DHAVE_QSORT_R_GLIBC=False \
+    -DHAVE___BUILTIN_BSWAP16=False \
+    -DHAVE___BUILTIN_BSWAP32=False \
+    -DHAVE___BUILTIN_BSWAP64=False \
+    .. 
+make -j${nproc} 
+make install 
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "windows"; ),
+    Platform("i686", "windows"; ),
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpostscriptlight", :libpostscriptlight),
+    LibraryProduct("libgmt", :libgmt),
+    ExecutableProduct("gmt", :gmt)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="LibCURL_jll", uuid="deac9b47-8bc7-5906-a0fe-35ac56dc84c0"); compat="7.73.0")
+    Dependency("NetCDF_jll", v"4.7.4")
```

Comment:
You're using the wrong netcdf version.  You should use the one corresponding to https://github.com/JuliaPackaging/Yggdrasil/blob/1ae78024d04123f2ee1e91736d61e2a807b004b6/N/NetCDF/NetCDF%40julia-1.6/build_tarballs.jl, which I guess is 400.702.402.  And you also must specify it as a compat

Reply:
thanks, just added that.

Reply:
With this change now this is working on two more platforms :slightly_smiling_face: